ubuntu20.04/generate-control-file

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -e
+
```

Comment:
Extra newline.

Reply:
removed this file to address deb generation as a separate PR